irqstat

```diff
@@ -303,6 +353,20 @@ def main(args):
     else:
         options.filter = []
 
+    # If file is specified, no iterations
+    if options.file1:
+        options.iterations = 1
+        options.batch=True
+
+    if options.file2:
+        if not options.file1:
+            print "XXX No file2 without file1"
```

Comment:
"--file2 requires --file1"

Reply:
Wow... How did I manage to miss that "XXX"?!?! :-P

Fixing. Thanks